src/components/button/test/raised/main.html

```diff
@@ -25,8 +25,8 @@
   </p>
 
   <p>
-    <button ion-button raised fab style="position: relative">FAB</button>
-    <button ion-button raised fab style="position: relative" class="activated">FAB</button>
+    <button ion-button raised ion-fab style="position: relative">FAB</button>
```

Comment:
First of all, thanks for the PR. Everything makes a lot of sense! just one change: I think you could remove this paragraph completely ^.